Cargo.toml

```diff
@@ -315,14 +320,18 @@ rstest = "0.17.0"
 rust-ini = "0.18.0"
 same-file = "1.0.6"
 selinux = "0.4"
+sha1 = "0.10.5"
+sha2 = "0.10.6"
+sha3 = "0.10.8"
 signal-hook = "0.3.15"
+sm3 = "0.4.2"
 smallvec = { version = "1.10", features = ["union"] }
 tempfile = "3.6.0"
 term_grid = "0.1.5"
 terminal_size = "0.2.6"
 textwrap = { version = "0.16.0", features = ["terminal_size"] }
 thiserror = "1.0"
-time = { version = "0.3" }
+time = { version = "=0.3.20" } ## maint: [2023-06-11; rivy] pinned to maintain MinSRV 1.64.0
```

Comment:
@sylvestre , this end-of-line comment gets blown out to starting at 100+ character position without the `align_comments = false`.

Reply:
you can probably make the comment shorter. remove the "maint" and the "maintain" 
or move it the line above

Reply:
IMO, it's better located on the line that it refers to so that they don't get separated with future change.
Ultimately, it'll be removable (and should be removed along with the `=` pin) when MinSRV increases.
And, no matter the length, if it's on the same line, `taplo` is pushing it to the right side, with an extremely wide area of white space between it and the code it refers too.